sharding-orchestration/sharding-orchestration-center/src/main/java/org/apache/shardingsphere/orchestration/center/instance/node/ConfigTreeNode.java

```diff
@@ -53,29 +53,29 @@
     private Set<ConfigTreeNode> childrenNodes;
     
     /**
-     * init tree.
+     * init config tree.
      * 
      * @param instanceKeys instance Key set
      * @param keySeparator key separator
      */
-    public void initTree(final Set<String> instanceKeys, final String keySeparator) {
-        initKeysRelationship(instanceKeys, keySeparator);
-        initTree(this, SHARDING_SPHERE_KEY_ROOT);
+    public void init(final Set<String> instanceKeys, final String keySeparator) {
+        initKeysRelationships(instanceKeys, keySeparator);
+        init(this, SHARDING_SPHERE_KEY_ROOT);
     }
     
-    private void initTree(final ConfigTreeNode parentNode, final String shardingSphereKey) {
+    private void init(final ConfigTreeNode parentNode, final String shardingSphereKey) {
```

Comment:
This should use another name, different from init().

Reply:
fix it, initTreeNode()